.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+
+---
+
+**Current Behavior**
+A clear and concise description of the behavior.
+
+**Input Code**
+- REPL or Repo link if applicable:
+
+```js
+var your => (code) => here;
+```
+**Expected behavior/code**
+A clear and concise description of what you expected to happen (or code).
+
+**Environment**
+- Node/npm version: [e.g. Node 8/npm 5]
```

Comment:
browser instead of node version?

Reply:
Yeah makes sense. How about `browser version OR Node version` in case someone has a bug with the build/linting process? Or should we redirect bug reports like this to the specific tool (webpack, loaders, etc.)?